sam_learning/learners/sam_learning.py

```diff
@@ -222,6 +224,18 @@ def construct_safe_actions(self) -> None:
         """Constructs the single-agent actions that are safe to execute."""
         pass
 
+    def handle_negative_preconditions(self) -> None:
+        """Function that handles ignoring negative preconditions."""
+        if not self.ignore_negative_preconditions:
+            return
+        for a in self.partial_domain.actions.values():
```

Comment:
We talked about me not liking variables with one letter that are not i in a for loop..